values.yaml

```diff
@@ -198,6 +198,18 @@ server:
   # If not set to true, Vault server will not be installed. See vault.mode in _helpers.tpl for implementation details
   enabled: true
 
+  # [Enterprise Only] This value refers to a Kubernetes secret that you have
+  # created that contains your enterprise license. If you are not using an
+  # enterprise image or if you plan to introduce the license key via another
+  # route, then set these fields to empty (""). Requires Vault Enterprise 1.8 or
+  # later.
+  enterpriseLicense:
+    # The name of the Kubernetes secret that holds the enterprise license. The
+    # secret must be in the same namespace that Vault is installed into.
+    secretName: ""
+    # The key within the Kubernetes secret that holds the enterprise license.
+    secretKey: ""
```

Comment:
Nit: should we have a reasonable default key name (like `license`)?

Reply:
Yeah, I think that makes sense. It'll make examples in the docs a little simpler too.

Reply:
Done in 95beeccfd6162ccf80a6bf97e64338fc5238f2f1